internal/publisher/cron.go

```diff
@@ -14,24 +14,24 @@ func ConfigureCron(ctx context.Context, logger *zap.Logger, config *model.Config
 	c := cron.New()
 
 	var err error
-	databaseClient, err := database.New(ctx, logger, &config.Database)
+	queueClient, err := queue.New(logger, ctx, &config.RabbitMq)
 	if err != nil {
 		return fmt.Errorf("Unexpected error when connecting to the database. %w", err)
 	}
-	defer databaseClient.CloseConnection(ctx)
-	service, err := NewService(logger, config, databaseClient)
+	defer queueClient.CloseConnection()
+	service, err := NewService(logger, config, queueClient)
 	if err != nil {
-		return fmt.Errorf("An error occurred when trying to instantiate the consumer service: %w", err)
+		return fmt.Errorf("An error occurred when trying to instantiate the publisher service: %w", err)
 	}
 	storyProcessing := func() {
-		err = service.processStories(ctx)
+		err = service.processStories()
```

Comment:
can this be removed from the anonymous function?

Reply:
Nope, in order to run in a cron you need an anonymous function